# /** ------分配控制器 ------ */

string                               au1ControllerName
uint32                               u4ObjectId
uint32                               u4PropertyNum
assign_controller_property[]         properties